pkg/config/config.go

```diff
@@ -135,7 +137,30 @@ func TestConfig() *Config {
 		},
 	}
 	conf.initLogger()
-	return conf
+	err := conf.initDisplay()
+	return conf, err
+}
+
+func (c *Config) initDisplay() error {
+	d := os.Getenv("DISPLAY")
+	if d != "" && strings.HasPrefix(d, ":") {
+		num, err := strconv.Atoi(d[1:])
+		if err == nil && num > 0 && num <= 2147483647 {
+			c.Display = d
+			return nil
+		}
+	}
+
+	if c.Display == "" {
+		c.Display = fmt.Sprintf(":%d", 10+rand.Intn(2147483637))
```

Comment:
does it actually work with maxInt? that's kinda. nuts.

Reply:
yeah, if using `-nolisten tcp`
https://stackoverflow.com/questions/12153552/how-high-do-x11-display-numbers-go